lib/absinthe/phase/schema/hydrate.ex

```diff
@@ -205,4 +205,16 @@ defmodule Absinthe.Phase.Schema.Hydrate do
 
     root
   end
+
+  def apply_hydration(root, result) do
```

Comment:
Should not this be deleted?

Reply:
No, providing a dedicated error message is better than simply getting a standard "no clause matching" error.